README.md

```diff
@@ -100,7 +110,7 @@ auth0:
 ```
 
 ## Transform
-The data received from the fetcher is being transformed using a transformation template, which is written as a go template and it outputs objects and relations.
+The data received from the fetcher is transformed into objects and relations using a transformation template, which is uses the go template syntax.
```

Comment:
Maybe we could add info about the pipeline before jumping to transform, similar to https://github.com/aserto-dev/aserto-docs/pull/146/files#r1310145608

Reply:
added a paragraph - thanks!